web/stream/service.go

```diff
@@ -61,27 +66,27 @@ func (s *Service[T]) SendTo(data *T, userIDs ...uint64) {
 }
 
 // Add adds a new stream for the given user to the service.
```

Comment:
I guess it is not longer only `the given user` but also a device... (although a device is perhaps a bit weird...) Would it make sense to say `Add adds a new stream to the service for the given identifier. The identifier may be a user ID or external application ID.` (not sure if the external application ID makes sense...)